eslint.config.mjs

```
@@ -0,0 +1,39 @@
+import globals from 'globals'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import js from '@eslint/js'
+import { FlatCompat } from '@eslint/eslintrc'
+
+const __filename = fileURLToPath(import.meta.url)
```

ESLint

[error] Insert `;`

(prettier/prettier)